src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const clickableImagesList = document.getElementById('thumbs');
+
+const largeImage = document.getElementById('largeImg');
+
+clickableImagesList.addEventListener('click', (eventOnClick) => {
+  // Prevent browser from navigating to the href
+  eventOnClick.preventDefault();
+
+  // Had some troubles with this.
+  /*
+    At first i've done this:
+    src = eventOnClick.target.getAttribite('src');
+  */
+  const src = eventOnClick.target.closest('.list-item__link').href;
```

Comment:
Ensure that the HTML structure has elements with the class 'list-item__link' and that these elements have an 'href' attribute. Otherwise, this line may cause errors if the structure does not match.